packages/front-end/components/Features/RuleList.tsx

```diff
@@ -46,6 +51,10 @@ export default function RuleList({
   experimentsMap: Map<string, ExperimentInterfaceStringDates>;
 }) {
   const { apiCall } = useAuth();
+  const [hideDisabled, setHideDisabled] = useLocalStorage(
+    `hide-disabled-rules-${environment}`,
```

Comment:
should this be per feature too? not sure if yes, just calling it out

Reply:
No, specifically was asked to make this per environment so they didn't have to keep clicking it.